cli/test/lib/tasks/verify_spec.js

```diff
@@ -53,6 +53,7 @@ context('lib/tasks/verify', () => {
     sinon.stub(xvfb, 'stop').resolves()
     sinon.stub(xvfb, 'isNeeded').returns(false)
     sinon.stub(Promise.prototype, 'delay').resolves()
+    sinon.stub(process, 'geteuid').returns(1000)
```

Comment:
well, if we have a test for "regular" user, we probably should have a test for root user?

Reply:
added a test for euid = 0